### DISCUSSION TITLE
Cannot specify icoUrl for custom chains?

### DISCUSSION THREAD

**Discussion by linghuccc** [wevm/wagmi#2578] [Q&A]
I was trying to add custom chains using rainbow kit by following their documentation in https://www.rainbowkit.com/docs/custom-chains. But an error was reported: iconUrl is not a valid parameter in Chain. Then I checked wagmi documentation on custom chains and found out that there is indeed no iconUrl in wagmi example. Does this mean we can not show any icon image for custom chains?

Dependency versions:
    "@rainbow-me/rainbowkit": "^1.0.1",
    "viem": "~0.3.35",
    "wagmi": "~1.0.5"

Please advise. Thank you.

**Comment by RexHung0302** [comment]
I got same promble, has anyone have any resolve?

**Comment by 6aldej** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by glitch-txs** [comment]
Hi we added this functionality in Web3Modal v3

https://docs.walletconnect.com/web3modal/react/options#chainimages
